terminology/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>server</artifactId>
+        <groupId>org.ehrbase.openehr</groupId>
+        <version>0.9.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>terminology</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.openEHR.archie</groupId>
+            <artifactId>openehr-rm</artifactId>
+            <version>9d65960</version>
```

Comment:
You should'n need the version since it is defined in the main.pom

Reply:
fixed with the new commit